includes/class-sensei-modules.php

```diff
@@ -1054,21 +1058,24 @@ public function calculate_user_module_progress( $user_id = 0, $module_id = 0, $c
 	}
 
 	/**
-	 * Register admin screen for ordering modules
+	 * Register admin pages related to modules.
 	 *
 	 * @since 1.8.0
-	 * @deprecated 4.0.0
 	 *
 	 * @return void
 	 */
 	public function register_modules_admin_menu_items() {
-		_deprecated_function( __METHOD__, '4.0.0' );
 
-		// add the modules link under the Course main menu
-		add_submenu_page( 'edit.php?post_type=course', __( 'Modules', 'sensei-lms' ), __( 'Modules', 'sensei-lms' ), 'manage_categories', 'edit-tags.php?taxonomy=module', '' );
+		// Register an admin page for module ordering.
+		add_submenu_page(
```

Comment:
I believe what @merkushin was suggesting [here](https://github.com/Automattic/sensei/pull/4646/files#r790807215) is that anyone calling this function will be surprised when they find that we've changed the implementation such that the _Modules_ menu is no longer added and the _Order Modules_ menu is hidden. It may be better to leave this one deprecated and add a new function to register the _Order Modules_ menu separately. WDYT?

Reply:
Now I understand the issue. Thanks for explaining.

It doesn't make much sense to me to preserve the old menu logic, but I guess it doesn't hurt. I've extracted the logic in a new method and re-deprecated the old method in 7f08a4e8f1cf6265eee38b1862f1e10997601ad8 and acee092473d24b891ed31bca03ce18e6b8e40f6f.